scripts/valhalla_build_config

```diff
@@ -136,45 +136,59 @@ config = {
   'service_limits': {
     'auto': {
       'max_distance': 5000000.0,
-      'max_locations': 20
+      'max_locations': 20,
+      'max_matrix_distance': 400000.0,
+      'max_matrix_locations': 50
     },
     'auto_shorter': {
       'max_distance': 5000000.0,
-      'max_locations': 20
+      'max_locations': 20,
+      'max_matrix_distance': 400000.0,
+      'max_matrix_locations': 50
     },
     'bus': {
       'max_distance': 5000000.0,
-      'max_locations': 50
+      'max_locations': 50,
+      'max_matrix_distance': 400000.0,
+      'max_matrix_locations': 50
     },
     'hov': {
       'max_distance': 5000000.0,
-      'max_locations': 20
+      'max_locations': 20,
+      'max_matrix_distance': 400000.0,
+      'max_matrix_locations': 50
     },
     'pedestrian': {
       'max_distance': 250000.0,
       'max_locations': 50,
+      'max_matrix_distance': 200000.0,
+      'max_matrix_locations': 50,
       'min_transit_walking_distance': 1,
       'max_transit_walking_distance': 10000
     },
     'bicycle': {
       'max_distance': 500000.0,
-      'max_locations': 50
+      'max_locations': 50,
+      'max_matrix_distance': 200000.0,
+      'max_matrix_locations': 50
     },
     'multimodal': {
       'max_distance': 500000.0,
-      'max_locations': 50
+      'max_locations': 50,
+      'max_matrix_distance': 200000.0,
+      'max_matrix_locations': 50
     },
     'transit': {
       'max_distance': 500000.0,
-      'max_locations': 50
+      'max_locations': 50,
+      'max_matrix_distance': 200000.0,
+      'max_matrix_locations': 50
     },
     'truck': {
       'max_distance': 5000000.0,
-      'max_locations': 20
-    },
-    'sources_to_targets': {
-      'max_distance': 200000.0,
-      'max_locations': 50
+      'max_locations': 20,
+      'max_matrix_distance': 400000.0,
+      'max_matrix_locations': 50
```

Comment:
Are these needed anymore? Seems this has been replaced by individual limits based on mode?